src/types/siteInfo.ts

```diff
@@ -26,6 +26,7 @@ export const SiteLaunchStatusObject = {
   Launched: "LAUNCHED",
   NotLaunched: "NOT_LAUNCHED",
   Launching: "LAUNCHING",
+  Failure: "FAILURE",
```

Comment:
Nit and non-blocking: `Failure` can be renamed to `Failed` and the value could be `FAILED` to make it consistent with other site launch status which are all past or ongoing tense of verb.

Reply:
sure correcsponging fe pr done b57172550a617e803b96f134c9589de6d85f676f